src/assets/CollectiblesController.test.ts

```diff
@@ -4,6 +4,7 @@ import HttpProvider from 'ethjs-provider-http';
 import { PreferencesController } from '../user/PreferencesController';
 import {
   NetworkController,
+  NetworkControllerMessenger,
```

Comment:
should prolly just reset this file and start over. Most of the diff is just formatting changes =(

Reply:
just dont use beforeEach to unsub... and we will b fine... maybe.